pkg/textparse/testdata.txt

```diff
@@ -525,4 +525,4 @@ prometheus_target_sync_length_seconds_count{scrape_job="prometheus"} 1
 prometheus_treecache_watcher_goroutines 0
 # HELP prometheus_treecache_zookeeper_failures_total The total number of ZooKeeper failures.
 # TYPE prometheus_treecache_zookeeper_failures_total counter
-prometheus_treecache_zookeeper_failures_total 0
```

Comment:
You still have a change here that isn't necessary. You can do `git checkout master -- pkg/textparse/testdata.txt` to revert it.